packages/landing/src/attribution.ts

```diff
@@ -119,7 +119,7 @@ export class MapAttribution {
     this.zoom += 1;
 
     const bbox = this.mapboxBoundToBbox(this.bounds, Config.map.tileMatrix);
-    const filtered = attr.filter(bbox, this.zoom, Config.map.dateRange.dateAfter, Config.map.dateRange.dateBefore);
+    const filtered = attr.filter(bbox, this.zoom, Config.map.filter.date.after, Config.map.filter.date.before);
```

Comment:
NIT: this function is starting to get too big, I think we should move it to a object too, it would be very easy to accidentally put `zoom, before, after` rather than `zoom, after, before` 

But can be done seperately

Reply:
Not a big change. I will do this now.

Reply:
I have make the change in #2709